nist_params.h

```diff
@@ -0,0 +1,108 @@
+#ifndef NIST_PARAM_H
+#define NIST_PARAM_H
+
+#include "params.h"
+
+
+#ifndef NIST_LEVEL
+#define NIST_LEVEL 0
+#endif
+
+/*
+ * NIST_LEVEL below does not make sense in term of security
```

Comment:
Curious: why is that? Is it worth explaining in more details?

Reply:
Yeah, all of the parameters are using SHA256, their security relied on hash security, so basically they are in the same security level to me. 
But the level I separate them by the number of signatures they can sign. 
I need a better name for this macro